common/app/Panes/redux/index.js

```diff
@@ -176,14 +171,22 @@ export default function createPanesAspects({ createPanesMap }) {
           ...state,
           panesMap,
           panes,
-          panesByName: panes.reduce((panes, { name }, index) => {
+          panesByName: panes.reduce((__panes, { name }, index) => {
```

Comment:
I can change this back from `__panes` to `panes` - I changed it at one point to try something, but no longer need it. Forgot about this.